ad2games-theme.gemspec

```diff
@@ -13,6 +13,5 @@ Gem::Specification.new do |s|
   s.files     = `git ls-files`.split("\n")
   s.require_paths = ['lib', 'assets', 'fonts']
 
-  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
Hmm. Really?

`sass-rails` already depends on ` s.add_dependency 'railties',        '>= 4.0.0', '< 5.1'`.

And seriously, who would use this gem with an ancient rails version anyway?